src/cls/IPM/Utils/Update/VersionBase.cls

```diff
@@ -0,0 +1,52 @@
+Class %IPM.Utils.Update.VersionBase [ Abstract ]
+{
+
+/// Returns dynamic array of update method names to apply in order that can be run on both primary and backup
+/// Example output list: ["methodA", "methodB", "methodC"]
+/// When mirror enabled: These steps will run on all members
+/// When mirror not enabled: These steps will be run on the instance
+ClassMethod GetOrderedMethodsAnyMirrorMember() As %Library.DynamicArray [ Abstract ]
```

Comment:
I would remove "AnyMirrorMember" and just call this GetOrderedMethods() since it implies its only relevant for mirrored instances.